lib/ParamedicAppUninstall.js

```diff
@@ -0,0 +1,68 @@
+var shelljs = require('shelljs');
+var path    = require('path');
+var fs      = require("fs");
+var logger  = require('./utils').logger;
+var util    = require('./utils').utilities;
+
+function ParamedicAppUninstall(appPath, platform) {
+    this.appPath  = appPath;
+    this.platform = platform;
+}
+
+ParamedicAppUninstall.prototype.uninstallApp = function(targetObj, app) {
+    if(!targetObj || !targetObj.target)
+        return;
+
+    switch(this.platform) {
+        case util.ANDROID:
+            this.uninstallAppAndroid(targetObj, app);
+            break;
+        case util.IOS:
+            this.uninstallAppIOS(targetObj, app);
+            break;
+        case util.WINDOWS:
+            this.uninstallAppWindows(targetObj, app);
+            break;
+        default:
+            break;
+    }
+}
+
+ParamedicAppUninstall.prototype.uninstallAppAndroid = function(targetObj, app) {
+    var uninstallCommand = "adb -s " + targetObj.target + " uninstall " + app;
+    logger.info("cordova-paramedic: Running command: " + uninstallCommand);
```

Comment:
Nitpick: you could factor out this shelljs stuff since the messages and result checking are the same for both platforms.